box/src/Box.ts

```diff
@@ -8,12 +8,8 @@ export class Box extends HTMLElement {
     const style = this.getAttribute('showcase-style') || '';
 
     this.innerHTML = /*html*/ `
-      <div class="${styles.wrapper}">
-        ${
-          hasCheckedBackground
-            ? `<div class="${styles.checkered} ${className}"></div>`
-            : ''
-        }
+      <div style="${style};${styles.wrapper}" class="${className}">
+        ${hasCheckedBackground ? `<div style="${styles.checkered}"></div>` : ''}
```

Comment:
`className` was previously on the inner DIV, but you moved it to the parent one
how is this related to TS migration? if you are fixijng some bugs, please create individual PRs with them and clear description why you do it